libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/ITriggerSelector.cs

```diff
@@ -13,6 +14,12 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive
     /// </summary>
     public interface ITriggerSelector
```

Comment:
>ITriggerSelector [](start = 21, length = 16)

We've been moving to abstract base classes instead of interfaces.  Should this be one?

Reply:
Probably, but I'm going to wait on this one.

---
In reply to: [360172438](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360172438) [](ancestors = 360172438)